.dockerignore

```diff
@@ -0,0 +1,13 @@
+.github
+.nyc_output
```

Comment:
removed from dockerignore. anything else I need to do?

Reply:
we recently moved to `c8` for code coverage instead of `nyc` https://github.com/digicatapult/wasp-event-service/blob/25b53161924e79ff23cb60f6ef4f5d069989e1ec/package.json#L12

Reply:
Removed reference to `nyc` is that it?